governance/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: inner-page-governance
+title: Governance
+permalink: /governance/
+includeTOC: true
+---
+
+## Who's behind Scala?
```

Comment:
It'd be a good idea to link to the core team and compiler team pages from this section

Reply:
Added a few sentences below, let me know it they are ok

Reply:
LGTM